### DISCUSSION TITLE
Configure directories and file types for 'recognize' to run in? Check status?

### DISCUSSION THREAD

**Discussion by dostnmatr** [nextcloud/recognize#803] [Q&A] [answered]
hi, I have a problem with recognize in a sense that I cannot comprehend what the plugin does or if it is still running correctly.
'Recognize' has been running for over two days on my machine (i5-8300, four cores). The number of files in queue is rising all the time and is currently at 27,300. I only enabled options to search for non-human elements in photos (object and landmark recognition) excluding all video and audio options. 
Where can I check what files and directories have already been worked through and which files are on queue?
How can I limit 'recognize' to certain directories and file types?
Why is the number of files in queue going up?
Thanks for your help!

**Comment by marcelklehr** [comment, marked as answer]
> Why is the number of files in queue going up?

The number is going up as new files are added to the queue. The queue is not constructed instantaneously but progressively.

> How can I limit 'recognize' to certain directories and file types?

You can limit recognize to certain directories by using .nomedia or .noimage tag files in directories that you don't want to use it on.

> Where can I check what files and directories have already been worked through and which files are on queue?

This information is currently not available in the User interface. You can see files that have been classified by the object detection via the admin-only tag "Tagged by recognize ...". Files classified by the face recognition should show up in the photos app. The admin settings overview should be much more helpful nowadays in troubleshooting issues, though.

**Comment by vbelloir** [comment]
Is  the .nomedia trick is recursive? 
I mean, are all subfolder also ignored?
